File: overview/rev1/guides/orders/php/simple/bank-account-create.php
<?php
$bank_account = $marketplace->bank_accounts->create(array(
"account_number" => "9900000001",
"name" => "Johann Bernoulli",
"routing_number" => "121000358",
"type" => "checking",
));

$bank_account->associateToCustomer($merchant);
?>

File: overview/rev1/guides/examples/java/bank-account-create.java
Map<String, Object> payload = new HashMap<String, Object>();
payload.put("account_number", "9900000001");
payload.put("name", "Johann Bernoulli");
payload.put("routing_number", "121000358");
payload.put("account_type", "checking");

BankAccount bankAccount = new BankAccount(payload);
try {
    bankAccount.save();
}
catch (HTTPError e) {}